versions.cfg

```diff
@@ -79,8 +81,38 @@ simplejson = 2.5.2
 six = 1.8.0
 cssmin = 0.2.0
 slimit = 0.8.1
+# stick to older ply 3.4, see https://github.com/dabeaz/ply/issues/82
 ply = 3.4
 
+# argparse is only needed for Python <2.7 but as zope toolkit 
+# pin an outdaed version be safe to pint to latest
+argparse = 1.4.0 
+argcomplete = 1.0.0
+
+# we dont use distribute (outdated fork of setuptools), buts its 
+# pinned in zope tookit. so update to latest legacy package, which is
+# a simple compatibility layer that installs Setuptools 0.7+
+distribute = 0.7.3
+
+# this is mentioned in zope.toolkit and outdated there - not sure if we need i
+# it at all
+z3c.recipe.sphinxdoc = 1.0.0
+
+# argparse is only needed for Python <2.7 but as zope toolkit 
+# pin an outdaed version be safe to pint to latest
+argparse = 1.4.0 
+argcomplete = 1.0.0
```

Comment:
dups, ops.
instead of a remove i would unset unused if it was inherited from somewhere, because otherwise buildouts inheriting from it may have bad side effects.